docs/web/docusaurus.config.js

```diff
@@ -44,8 +44,11 @@ const config = {
         theme: {
           customCss: require.resolve('./src/css/custom.css'),
         },
-        gtag: {
-          trackingID: 'GTM-NGPJM9R',
+        // gtag: {
```

Comment:
Can this commented code be deleted? It would appear that it is googleTagManager versus gtag.